configure.ac

```diff
@@ -360,7 +360,7 @@ case $host in
 
      AX_CHECK_LINK_FLAG([[-Wl,-headerpad_max_install_names]], [LDFLAGS="$LDFLAGS -Wl,-headerpad_max_install_names"])
      CPPFLAGS="$CPPFLAGS -DMAC_OSX"
-     OBJCXXFLAGS="$CXXFLAGS"
+     OBJCXXFLAGS="-std=c++11 $CXXFLAGS"
```

Comment:
Instead of directly adding -lcurl in *C*FLAGS, I'd add a check for the library in configure.ac. Regarding this branch everything is ok.